media-libs/speex/speex-1.2.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit autotools flag-o-matic multilib-minimal
+
+MY_P=${P/_} ; MY_P=${MY_P/_p/.}
+
+DESCRIPTION="Audio compression format designed for speech"
+HOMEPAGE="http://www.speex.org/"
+SRC_URI="http://downloads.xiph.org/releases/speex/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ia64 ~mips ~ppc ~ppc64 ~sh ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
+IUSE="static-libs utils +vbr armv5te cpu_flags_x86_sse"
+
+RDEPEND="
+	utils? (
+		media-libs/libogg:=
+		media-libs/speexdsp
+	)"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+S=${WORKDIR}/${MY_P}
+
+PATCHES=( "${FILESDIR}/${P}-configure.patch" )
+
+src_prepare() {
+	default
+
+	sed -i \
+		-e 's:noinst_PROGRAMS:check_PROGRAMS:' \
+		libspeex/Makefile.am || die
+
+	eautoreconf
+}
+
+multilib_src_configure() {
+	append-lfs-flags
+
+	# Can also be configured without floating point
+	# --enable-fixed-point
+	ECONF_SOURCE="${S}" econf \
+		$(use_enable static-libs static) \
+		$(use_enable (arm && !armv5te) arm4-asm) \
```

Comment:
This causes a syntax error for me. I believe the first argument of ``use_enable`` [should be](https://projects.gentoo.org/pms/latest/pms.html#x1-14100011.3.3.12) a USE flag.

Perhaps this could be replaced with ``$(use_enable !armv5te arm4-asm)``? The ``armv5te`` USE flag is masked on all non-arm profiles. Maybe @SoapGentoo can offer a better solution?